farmbot_core/lib/farmbot_core/asset/command.ex

```diff
@@ -6,17 +6,36 @@ defmodule FarmbotCore.Asset.Command do
   alias FarmbotCore.{
     Asset,
     Asset.Repo,
-    Asset.Device
+    Asset.Device,
+    Asset.FbosConfig
   }
 
-  @type kind :: Device | FbosConfig | FwConfig
-  @type return_type ::
-          Device.t() | FbosConfig.t() | FwConfig.t()
+  @type kind :: Device | FbosConfig | FwConfig | FarmwareEnv
```

Comment:
`FwConfig` and `FarmwareEnv` are not aliased. Check for compiler warnings about this.